nimare/utils.py

```diff
@@ -33,53 +32,63 @@ def get_resource_path():
 def get_template(space="mni152_2mm", mask=None):
     """Load template file.
 
+    .. versionchanged:: 0.0.11
+        Remove the mask="gm" option.
+        Replace the nilearn templates with ones downloaded directly from TemplateFlow.
+
     Parameters
     ----------
     space : {'mni152_1mm', 'mni152_2mm', 'ale_2mm'}, optional
-        Template to load. Default is 'mni152_1mm'.
-    mask : {None, 'brain', 'gm'}, optional
-        Whether to return the raw template (None), a brain mask ('brain'), or
-        a gray-matter mask ('gm'). Default is None.
+        Template to load. Default is 'mni152_2mm'.
+        The options are:
+
+            -   mni152_1mm: The MNI152NLin6Asym template at 1mm3 resolution,
+                downloaded from TemplateFlow. The shape of this template is X voxels.
```

Comment:
```suggestion
                downloaded from TemplateFlow. The shape of this template is 182x218x182 voxels.
```

I don't want to wait around for CI to pass again, so once it passes I'm going to commit this change and force the merge through.